sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLSecurityProfileDAO.java

```diff
@@ -207,8 +201,8 @@ public int count() throws IOException {
   @Cacheable(cacheName = "securityProfileCache", keyGenerator = @KeyGenerator(name = "HashCodeCacheKeyGenerator", properties = {
       @Property(name = "includeMethod", value = "false"), @Property(name = "includeParameterTypes", value = "false") }))
   public SecurityProfile get(long id) throws IOException {
-    List results = template.query(PROFILE_SELECT_BY_ID, new Object[] { id }, new SecurityProfileMapper());
-    SecurityProfile sp = results.size() > 0 ? (SecurityProfile) results.get(0) : null;
+    List<SecurityProfile> results = template.query(PROFILE_SELECT_BY_ID, new Object[] { id }, new SecurityProfileMapper());
+    SecurityProfile sp = results.size() > 0 ? results.get(0) : null;
```

Comment:
This pattern is in a lot of places... Can we add `if (results.size() > 1) throw new IllegalStateException("...")` before this line?